pkg/controllers/cloud/networkpolicy_controller.go

```diff
@@ -103,14 +102,23 @@ type NetworkPolicyReconciler struct {
 	cloudResponse chan *securityGroupStatus
 
 	// syncedWithCloud is true if controller has synchronized with cloud at least once.
-	syncedWithCloud bool
+	syncedWithCloud   bool
+	cloudSyncInterval int64
+	lastSyncTime      int64
+
 	// Bookmark events received prior to sync with the cloud.
 	bookmarkCnt int
 
 	// localRequest sends and receives network policy requests from local stack.
 	localRequest chan watch.Event
 }
 
+func (r *NetworkPolicyReconciler) SetCloudSyncInterval(controllerConfig *config.ControllerConfig) {
+	r.cloudSyncInterval = controllerConfig.CloudSyncInterval
+	r.lastSyncTime = time.Now().Unix()
+	r.Log.V(1).Info("Updated the CloudSyncInterval")
```

Comment:
Change the log leve to Info 
```suggestion
	r.Log.Info("Updated the CloudSyncInterval")
```

Reply:
Is this function access outside the cloud package